jenkins/build-opm-module.sh

```diff
@@ -151,18 +150,10 @@ function clone_module {
   if [ "$1" == "libecl" ]
   then
     git remote add origin https://github.com/equinor/$1
-    if [[ "$2" == 7abee4f1* ]]
```

Comment:
In theory one should be able to specify a commit with "libec=12a6fbbb1...", but in practice that fails because github does not allow us to git fetch a particular commit, only branches and tags. To get a particular commit one must do as in the code that is removed here (get master with full depth, then checkout the particular commit). Perhaps it is worth preserving that?

What about simply removing this if, always doing what is now lines 156-158 for libecl? That way we could eat our cake and have it, too. We could do the same for all repos, but I suspect the shallow fetch is cheaper both in time, network activity and disk space, and it is not likely to be useful.